base/markdown/Julia/interp.jl

```diff
@@ -39,7 +39,7 @@ end
 
 toexpr(x) = x
 
-toexpr(xs::Vector{Any}) = Expr(:cell1d, map(toexpr, xs)...)
+toexpr(xs::Vector{Any}) = Expr(:vectany, map(toexpr, xs)...)
```

Comment:
Well, it's really easy to rename, so I figured it would be simpler to do this is small steps. (I actually tried to remove the code parsing the deprecated syntax, but it's hard when you don't understand the code...)